src/plugins/intel_gna/tests/functional/Import_export_tests/backward_compatibility/backward_compatibility.cpp

```diff
@@ -0,0 +1,229 @@
+        auto read = std::make_shared<ReadValue>(initial_state_node, var);
+        auto mul = std::make_shared<Multiply>(split_1_reshape, read);
+        auto assign = std::make_shared<Assign>(mul, var);
+        auto relu_3 = std::make_shared<Relu>(mul);
+
+        ov::SinkVector sinks = {assign};
+        ov::ResultVector results;
+        results.emplace_back(std::make_shared<Result>(relu_2));
+        results.emplace_back(std::make_shared<Result>(relu_3));
+
+        function = std::make_shared<ov::Model>(results, sinks, inputs, "universal_export_model");
+    }
+
+    std::map<std::string, std::string> exportConfiguration;
+    std::map<std::string, std::string> importConfiguration;
+    std::string m_export_model_name;
+};
+
+class BackwardCompatibilityLegacy : public BackwardCompatibility {
+protected:
+    void SetUp() override {
+        ov::element::Type prc = ov::element::undefined;
+        std::tie(prc, targetDevice, m_export_model_name, exportConfiguration, importConfiguration) = this->GetParam();
+        ov::Shape input_shape{1, 336};
+
+        auto param = std::make_shared<Parameter>(prc, input_shape);
+        auto const_eltwise = std::make_shared<Constant>(prc, input_shape, std::vector<float>{-1});
+        auto mul = std::make_shared<Multiply>(param, const_eltwise);
+
+        ov::ResultVector results{std::make_shared<Result>(mul)};
+
+        function = std::make_shared<ov::Model>(results, ov::ParameterVector{param}, "ExportBackwordCompatibility");
+    }
+};
+
+TEST_P(BackwardCompatibility, smoke_BackwardCompatibility) {
+    Run();
+}
+
+TEST_P(BackwardCompatibilityLegacy, smoke_BackwardCompatibility) {
+    Run();
+}
+
+const std::vector<ov::element::Type> input_precisions = {ov::element::f32, ov::element::f16};
+
+const std::vector<std::map<std::string, std::string>> export_configs_legacy = {
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}}};
+
+const std::vector<std::map<std::string, std::string>> import_configs_legacy = {
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}},
+};
+
+const std::vector<std::map<std::string, std::string>> export_configs = {
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}, {"GNA_SCALE_FACTOR_1", "327.67"}}};
+
+const std::vector<std::map<std::string, std::string>> import_configs = {
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}, {"GNA_SCALE_FACTOR_1", "327.67"}}};
+
+const std::vector<std::string> export_models_legacy = {"export2dot1.blob",
+                                                       "export2dot2.blob",
+                                                       "export2dot3.blob",
+                                                       "export2dot4.blob",
+                                                       "export2dot5.blob"};
+const std::vector<std::string> export_models = {"export2dot6.blob",
```

Comment:
Does it make sense to use new config API for models in `export_models`?

Reply:
Do you mean to set scale factors?

Reply:
> Do you mean to set scale factors?

I mean all config properties

Reply:
We agreed that this needs to be done for all the tests and as separate change